packages/cli/src/lib/defaults/infra-modules/eth-ens-ipfs/docker-compose.yaml

```diff
@@ -1,7 +1,6 @@
 version: '3'
 services:
   ganache:
-    platform: linux/x86_64
```

Comment:
Yes this seems like the correct approach, remove the specific platform specification, it's unneeded. This is because the docker image being used (`node:16-alpine`, see `./eth/Dockerfile`), is a multi-platform image. This means that docker will automatically detect what CPU architecture it is running within (amd64, arm, etc) and download the correct build of the image from docker hub: https://hub.docker.com/_/node/tags

Reply:
Thank you for the detailed explanation!